### PR TITLE
Release 5 0 x

### PR DESCRIPTION
**Pull request by hkollmann**
fix URL

### CODE REVIEW

File: documentation/manual/source/_theme.indigo/layout.html [hunk 1]
```diff
@@ -32,25 +32,20 @@
...
-        <!--
         <li>
-          <a class="" href="http://qooxdoo.org/">home</a>
+          <a class="" href="/demos.html">demos</a>
         </li>
-        -->
         <li>
-          <a href="http://news.qooxdoo.org">blog</a>
+          <a class="" href="https://github.com/qooxdoo/qooxdoo/releases">downloads</a>
         </li>
```
Comment (johnspackman) [inline documentation/manual/source/_theme.indigo/layout.html]: Blog should stay, it's now at https://medium.com/qooxdoo

File: documentation/manual/source/_theme.indigo/layout.html [hunk 2]
```diff
@@ -32,25 +32,20 @@
...
         <li>
-          <a class="" href="http://qooxdoo.org/demos">demos</a>
+          <a class="active" href="/current">docs</a>
         </li>
         <li>
-          <a class="" href="http://qooxdoo.org/downloads">downloads</a>
```
Comment (johnspackman) [inline documentation/manual/source/_theme.indigo/layout.html]: Why is downloads no longer in the navigation?

**Comment by johnspackman** [comment]
Im wondering if Im missing something here - where does this appear on the site at the moment?

In any case, this seems to do more than just correct a couple of urls, please can you explain further?

File: documentation/manual/source/_theme.indigo/layout.html (hunk 1 above)
Comment (hkollmann) [inline documentation/manual/source/_theme.indigo/layout.html]: See comment in  #9109

**Pull request merged**